### PR TITLE
[doc] Updated the compilation and deployment chapter in the documentation

### PR DESCRIPTION
**Pull request by hanfengcan**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.
增加使用supervisord遇到句柄数问题的解决方法

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/install/install-deploy.md [hunk 1]
```diff
@@ -486,3 +486,11 @@ Broker is a stateless process that can be started or stopped at will. Of course,
    For Example: ulimit -n 65536; this command set file descriptor to 65536.
 
    After starting BE process, you can use **cat /proc/$pid/limits** to see the actual limit of process.
+
+   if use supervisord, try to modify `minfds` in supervisord.conf
```
Comment (jackwener) [inline docs/en/install/install-deploy.md:490]:
```suggestion
   if use `supervisord`, try to modify `minfds` in supervisord.conf
```

**Review by jackwener — DISMISSED** [review]
LGTM, cc @hf200012

**Comment by jackwener** [comment]
Thanks for your contribution❤😀

**Comment by morrySnow** [comment]
title follows Commit Format Specification is better: https://doris.apache.org/zh-CN/community/how-to-contribute/commit-format-specification.html

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**